core/src/main/scala/cats/package.scala

```diff
@@ -77,6 +77,18 @@ package object cats {
       override def isEmpty[A](fa: Id[A]): Boolean = false
   }
 
+  /**
+   * Witness for: Id[A] <-> Unit => A
```

Comment:
we could add something similar for `Eval[A]` potentially.

Reply:
+1 , Since this has been blocking 1.2.0 release, I propose we tackle that in a separate PR
I added an issue https://github.com/typelevel/cats/issues/2320